angular/src/lib/select/select.component.ts

```diff
@@ -115,10 +114,9 @@ export class SelectChange {
 
       <div
         class="md-list md-list--vertical"
-        [style.height]="this.options.length > 8 ? scrollHeight : 'auto'"
-        [style.max-height]="scrollHeight"
+        [style.max-height]="this.options && this.options.length > 8 ? scrollHeight : '350px'"
```

Comment:
350px?  where did that come from?

Reply:
A default height was needed to avoid a large list showing

Reply:
Can we use rem instead of px to make sure it is a relative measure?